spec/requests/api/v1/inventory_list_items_controller_spec.rb

```diff
@@ -0,0 +1,351 @@
+    it 'Response with correct inventory list items, default per page' do
+      hash_body = nil
+      get api_v1_team_inventory_column_list_items_path(
+        team_id: @teams.first.id,
+        inventory_id: @teams.first.repositories.first.id,
+        column_id: @list_column.id
+      ), headers: @valid_headers
+      expect { hash_body = json }.not_to raise_exception
+      expect(hash_body[:data]).to match(
+        ActiveModelSerializers::SerializableResource
+          .new(@list_column.repository_list_items.limit(10),
+               each_serializer: Api::V1::InventoryListItemSerializer)
+          .as_json[:data]
+      )
+    end
+
+    it 'When invalid request, user in not member of the team' do
+      hash_body = nil
+      get api_v1_team_inventory_column_list_items_path(
+        team_id: @wrong_inventory.team.id,
+        inventory_id: @wrong_inventory.id,
+        column_id: @wrong_inventory.repository_columns.first.id
+      ), headers: @valid_headers
+      expect(response).to have_http_status(403)
+      expect { hash_body = json }.not_to raise_exception
+      expect(hash_body).to match({})
+    end
+
+    it 'When invalid request, non existing inventory' do
+      hash_body = nil
+      get api_v1_team_inventory_column_list_items_path(
+        team_id: @teams.first.id,
+        inventory_id: 123,
+        column_id: 999
+      ), headers: @valid_headers
+      expect(response).to have_http_status(404)
+      expect { hash_body = json }.not_to raise_exception
+      expect(hash_body).to match({})
+    end
+
+    it 'When invalid request, repository from another team' do
+      hash_body = nil
+      get api_v1_team_inventory_column_list_items_path(
+        team_id: @teams.first.id,
+        inventory_id: @wrong_inventory.id,
+        column_id: @list_column.id
+      ), headers: @valid_headers
+      expect(response).to have_http_status(404)
+      expect { hash_body = json }.not_to raise_exception
+      expect(hash_body).to match({})
+    end
+  end
+
+  describe 'GET inventory_list_items, #show' do
+    it 'Response with correct inventory list item' do
+      hash_body = nil
+      get api_v1_team_inventory_column_list_item_path(
+        id: @list_column.repository_list_items.first.id,
+        team_id: @teams.first.id,
+        inventory_id: @teams.first.repositories.first.id,
+        column_id: @list_column.id
+      ), headers: @valid_headers
+      expect { hash_body = json }.not_to raise_exception
+      expect(hash_body[:data]).to match(
+        ActiveModelSerializers::SerializableResource
+          .new(@list_column.repository_list_items.first,
+               serializer: Api::V1::InventoryListItemSerializer)
+          .as_json[:data]
+      )
+    end
+
+    it 'When invalid request, non existing list item' do
+      hash_body = nil
+      get api_v1_team_inventory_column_list_item_path(
+        id: 999,
+        team_id: @teams.first.id,
+        inventory_id: 123,
```

Comment:
Shouldn't `inventory_id` and `column_id` be same as in the previous test (e.g. non-random), since we're just testing the ID of the list item?